learning_assistant/api.py

```diff
@@ -109,9 +108,15 @@ def render_prompt_template(request, user_id, course_run_id, unit_usage_key, cour
     """
     unit_content = ''
 
-    course_run_key = CourseKey.from_string(course_run_id)
-    if unit_usage_key and course_content_enabled(course_run_key):
-        _, unit_content = get_block_content(request, user_id, course_run_id, unit_usage_key)
+    if unit_usage_key:
+        try:
+            _, unit_content = get_block_content(request, user_id, course_run_id, unit_usage_key)
+        except InvalidKeyError:
+            log.info(
```

Comment:
Are you sure we want to log `info` here?
Why not `error` or `warning`?

Reply:
Warning might be more appropriate. I wasn't sure about logging an error because I wasn't sure if it would impact any DD monitors watching error rates in logs (if those exist).

The reason I initially chose info was because it's something for us to keep track of to better understand the scope of this and when it occurs, but it's not an indication that the request won't succeed.